cfy_manager/components/globals.py

```diff
@@ -91,10 +93,86 @@ def _apply_forced_settings():
         config[POSTGRESQL_CLIENT][SSL_ENABLED] = True
 
 
+class Listener(TypedDict, total=False):
```

Comment:
Any reason for preferring a typeddict to a dataclass?

Reply:
I think it'll still need to be able to be jsonified? Won't it?

Reply:
I don't think so- jinja2 can use python objects, and I don't think it gets dumped out anywhere else that craves json, unless I forgot something?